nemo/collections/asr/models/rnnt_models.py

```diff
@@ -815,7 +816,7 @@ def predict_step(self, batch, batch_idx, dataloader_idx=0):
         del signal
 
         best_hyp_text = self.decoding.rnnt_decoder_predictions_tensor(
-            encoder_output=encoded, encoded_lengths=encoded_len, return_hypotheses=False
+            encoder_output=encoded, encoded_lengths=encoded_len, return_hypotheses=True
```

Comment:
This should be handled for all models later gracefully as we return hypotheses always now.